ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -252,29 +252,39 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 // it would completely clog up any EF actions. We've used manually created migrations instead.
             });
 
+            modelBuilder.Entity<PatientDetails>(entity =>
+            {
+                entity.HasKey(pd => pd.NotificationId);
+
+                entity.HasOne(pd => pd.PostcodeLookup)
+                    .WithMany()
+                    .HasForeignKey(ns => ns.PostcodeToLookup);
+
+                entity.ToTable("Patients");
+            });
+
+            modelBuilder.Entity<HospitalDetails>(entity =>
+            {
+                entity.HasKey(pd => pd.NotificationId);
+
+                entity.HasOne(hd => hd.CaseManager);
```

Comment:
Does this need a `WithMany()`?

Reply:
I'm not sure what the parameterless `WithMany` call actually does. The docs say:

> Note that calling this method with no parameters will explicitly configure this side of the relationship to use no navigation property, even if such a property exists on the entity type. If the navigation property is to be used, then it must be specified.

...but they don't say what happens if you leave it out. My guess is that it doesn't really have any impact on behaviour.

That said, it would probably be better to add the call, as it makes the intent clearer.